lib/nolibc/musl-imported/src/string/strstr.c

```diff
@@ -0,0 +1,156 @@
+/* taken from musl */
+
+#include <string.h>
+#include <stdint.h>
+
+static char *twobyte_strstr(const unsigned char *h, const unsigned char *n)
+{
+	uint16_t nw = n[0]<<8 | n[1], hw = h[0]<<8 | h[1];
+	for (h++; *h && hw != nw; hw = hw<<8 | *++h);
```

Comment:
Hi @mogasergiu ,

Can you please fix the linting problem?

Reply:
Yeah this does look like something worth aligning to our coding style standards :+1:.